src/Symfony/Component/Serializer/Normalizer/SanitizeDenormalizer.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Normalizer;
+
+use Symfony\Component\HtmlSanitizer\HtmlSanitizerInterface;
+use Symfony\Component\Serializer\Attribute\Sanitize;
+use Symfony\Component\Serializer\Exception\BadMethodCallException;
+use Symfony\Component\Serializer\Exception\InvalidArgumentException;
+use Symfony\Component\Serializer\Exception\LogicException;
+
+/**
+ * Denormalizer that sanitizes string properties marked with the #[Sanitize] attribute.
+ *
+ * @author Mohamed Senoussi <lesfootix@gmail.com>
+ */
+final class SanitizeDenormalizer implements DenormalizerInterface
+{
+    use DenormalizerAwareTrait;
```

Comment:
using this trait without implementing `DenormalizerAwareInterface` does not make sense